src/routes/index.ts

```diff
@@ -286,7 +287,7 @@ router.route(config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL)
     navigation.hasOverseasEntity
   )
   .get(confirmOverseasEntityDetails.get)
-  .post(confirmOverseasEntityDetails.post);
+  .post(companyAuthentication, confirmOverseasEntityDetails.post);
```

Comment:
Are we going to add it to all the routes after the auth code page?

Reply:
Should it not be after the confirmation page? i.e. user enters number; confirms they have right one; then asked for auth code.

Reply:
I think it should. What if we get navigation wrong anywhere. Authentication middleware is everywhere too.

Reply:
@AndreKramer7654 , have added this everywhere as its easier to understand from the code and avoiding the risk of navigation checks being wrong, this will result in calls to web-security-node from all the pages.

Reply:
I'm hoping it calls the web-security SDK but then finds that we are already authenticated without calling a remote API or link so that any overhead is minimal.